<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SmartPreviewDialog</class>
 <widget class="QDialog" name="SmartPreviewDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>976</width>
    <height>737</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Print Preview</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../../home/src/res.qrc">
    <normaloff>:/images/Printer.png</normaloff>:/images/Printer.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="SmartPreviewWidget" name="previewWidget" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>3</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QToolButton" name="cmdZoomIn">
       <property name="font">
        <font>
         <pointsize>24</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>+</string>
       </property>
       <property name="iconSize">
        <size>
         <width>64</width>
         <height>64</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="cmdZoomOut">
       <property name="font">
        <font>
         <pointsize>24</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>-</string>
       </property>
       <property name="iconSize">
        <size>
         <width>64</width>
         <height>64</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QToolButton" name="cmdPrint">
       <property name="font">
        <font>
         <pointsize>14</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Print</string>
       </property>
       <property name="icon">
        <iconset resource="../../home/src/res.qrc">
         <normaloff>:/images/Printer.png</normaloff>:/images/Printer.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>34</width>
         <height>32</height>
        </size>
       </property>
       <property name="toolButtonStyle">
        <enum>Qt::ToolButtonTextUnderIcon</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>SmartPreviewWidget</class>
   <extends>QWidget</extends>
   <header>smartpreviewwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../home/src/res.qrc"/>
 </resources>
 <connections/>
</ui>
